wardrobe/money/src/lib.rs

```diff
@@ -95,8 +95,8 @@ impl Accumulator for ImbalancedFundsAccumulator {
         0
     }
 
-    fn key_path(_: Self::ValueType) -> &'static str {
-        "imbalanc"
+    fn key_path(_: Self::ValueType) -> &'static [u8] {
+        b"imbalanc"
```

Comment:
imbalance with an e

Previously I was using `[u8; 8]` and that's why the e was missing, but we can now use the full word.

Reply:
addressed in 3de2e96d5718e1714b17eaae8b87ce8efe0f6da6